src/poly1305.rs

```diff
@@ -57,6 +141,26 @@ pub fn sign(key: Key, msg: &[u8], tag: &mut Tag) {
     ctx.sign(tag)
 }
 
+#[inline]
+fn check_block_state_layout() {
```

Comment:
This can be done as a test, right? I don't think we need to check this at runtime outside of tests.